src/components/header/NavigationLink.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, ReactNode } from 'react';
+
+import { Language, useI18n } from '@smg-automotive/i18n-pkg';
+import { Badge } from '@chakra-ui/react';
```

Comment:
You can use our component `Badge` instead importing it through '@chakra-ui/react' Thanks  🌸